package.json

```diff
@@ -6,16 +6,17 @@
   "scripts": {
     "test": "jest",
     "lint": "eslint .",
-    "serve": "netlify-lambda serve functions",
-    "build": "netlify-lambda build functions"
+    "serve": "netlify-lambda serve functions --config ./functions/webpack.functions.js",
+    "build": "netlify-lambda build functions --config ./functions/webpack.functions.js"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "dependencies": {
     "axios": "0.18.1",
     "dotenv": "8.0.0",
-    "netlify-lambda": "1.4.13"
+    "netlify-lambda": "1.4.13",
+    "webpack": "^4.41.2"
```

Comment:
This package should be added as a dev dependency since it is only used at build time.